apps/web/templates/about.html

```diff
@@ -0,0 +1,213 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block menu %}
+<!--==========================
+    Menu
+============================-->
+<nav id="nav-menu-container">
+  <ul class="nav-menu">
+    <li><a href="#body">Home</a></li>
+    <li class="menu-has-children menu-active"><a href="">SBG-RJ</a>
+      <ul>
+        <li><a href="#">Sobre</a></li>
+        <li><a href="#">Eventos</a></li>
+        <li><a href="#">Notícias</a></li>
+        <li><a href="#">Produtos</a></li>
+      </ul>
+    </li>
+    <li><a href="#">Publicações</a></li>
+    <li><a href="#">Galeria</a></li>
+    <li><a href="#">Regionais</a></li>
+    <li><a href="#">Contato</a></li>
+  </ul>
+</nav><!-- #nav-menu-container -->
+{% endblock menu %}
+
+{% block intro %}
+<!--==========================
+    Intro Section
+============================-->
+<section id="intro">
+  <div class="intro-content">
+    <h2>SOCIEDADE BRASILEIRA DE GEOLOGIA – NÚCLEO RIO DE JANEIRO</h2>
+    <div class="container">
+      <img src="../static/img/geology-icon.png"></img>
```

Comment:
Utilize as tags do template para carregar os arquivos staticos, media e urls.

Isso pode gerar problemas no deploy pq os arquivos staticos nem sempre ficam na pasta do projeto para que o `nginx`, ou oq formos utilizar, tenha acesso e consiga disponibilizar essas imagens. Dessa forma ele pega pelas configurações independente de onde definirmos.

```suggestion
      <img src="{% static 'img/geology-icon.png' %}" alt="Imagem do ..." title="Tal lugar" ></img>
```

- É bacana sempre pensar no pessoal que não enxerga e precisa de leitores de tela, o valor em `alt` ajuda a descrever a imagem para eles e a rankear melhor a página.
- Coloque sempre um `title` nas imagens, assim caso elas não sejam carregadas ele aparecerá no local (ou quando vc passar o mouse encima).